src/service-override/views.ts

```diff
@@ -241,7 +256,7 @@ function registerEditorPane (options: EditorPanelOption): { disposable: IDisposa
     private description: Label = options.name
     private dirty: boolean = false
 
-    constructor (public override readonly closeHandler?: IEditorCloseHandler) {
+    constructor (public override readonly closeHandler?: IEditorCloseHandler, public baseInput?: IResourceEditorInput | ITextResourceEditorInput | undefined, private internalEditorId?: string | undefined) {
```

Comment:
`baseInput` is needed for the `resource` call which VSCode uses to deduplicate editors based on resources (I think). This is the description:

<img width="1040" alt="image" src="https://github.com/CodinGame/monaco-vscode-api/assets/587016/45f3ba79-13a6-4ee9-9d51-fc1d85f41be6">